composer.json

```diff
@@ -50,7 +50,7 @@
         "oat-sa/extension-tao-outcomekeyvalue": "6.1.1",
         "oat-sa/extension-tao-outcomelti": "5.0.2",
         "oat-sa/extension-tao-delivery": "15.5.2",
-        "oat-sa/extension-tao-delivery-rdf": "14.7.2",
+        "oat-sa/extension-tao-delivery-rdf": "dev-fix/ADF-1013/stored-theme-is-not-selected-for-delivery as 14.7.3",
```

Comment:
Has to be updated once the PR https://github.com/oat-sa/extension-tao-delivery-rdf/pull/422 is merged.